backend/lambda/database_calls.py

```diff
@@ -81,7 +81,14 @@ def record_call(self, existing_id, base_id, user_id, stage, status_code, respons
                     )
                     cur.execute(query)
         except Exception as e:
-            print("ERROR: Unexpected error: Could not commit to database instance.")
+            error = dict(
+                severity="ERROR",
```

Comment:
If we wanna keep things DRY this `error` could be a Subclass of `Error` and you could add a serialize function for `json`. Or this could just be a constant object.

Reply:
A fair point, although I'm not sure it will make things any more simplified. It would just remove the severity line but the message and component would still need to be set.

Reply:
I'm going to keep it as is for now, then if we use this in a lot of other places I'll refactor it